src/gui/wizard/owncloudadvancedsetuppage.ui

```diff
@@ -377,6 +380,46 @@
           </item>
          </layout>
         </item>
+        <item>
+         <layout class="QHBoxLayout" name="horizontalLayout_4">
+          <item>
+           <widget class="QRadioButton" name="rPlaceholderSync">
+            <property name="sizePolicy">
+             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
+              <horstretch>0</horstretch>
+              <verstretch>0</verstretch>
+             </sizepolicy>
+            </property>
+            <property name="text">
+             <string/>
+            </property>
+            <property name="checkable">
+             <bool>false</bool>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QPushButton" name="bPlaceholderSync">
```

Comment:
Why is it a QPushButton?

The "choose what to sync" option is a button because clicking on it allow to choose what to sync.
But here there is no reason to push the button.

I'd say we should keep a normal checkbox.